rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -587,6 +593,14 @@ private Integer addFederatorPublicKeyMultikey(boolean dryRun, BtcECKey btcKey, E
         return FederationChangeResponseCode.SUCCESSFUL.getCode();
     }
 
+    protected int commitFederation(boolean dryRun, Keccak256 hash, BridgeEventLogger eventLogger) {
+        if (!activations.isActive(ConsensusRule.RSKIP419)) {
+            return commitFederationPreRSKIP419(dryRun, hash, eventLogger);
+        }
+
+        return commitFederationPostRSKIP419(dryRun, hash, eventLogger);
```

Comment:
I prefer to call them `legacyCommitFederation` and `commitFederation`

Makes it clearer to me at least

Reply:
for me its clearer to have the rskip, but i'll change it. No problem

Reply:
Done